server/api/tests/workspace.py

```diff
@@ -1,4 +1,62 @@
 from django.urls import reverse
 from rest_framework.test import APITestCase
 from rest_framework import status
-from ..models import Workspace, WorkspaceMember
\ No newline at end of file
+from rest_framework_simplejwt.tokens import RefreshToken
+from ..models import Workspace, WorkspaceMember, User
+
+class AddUserTests(APITestCase):
```

Comment:
Just nitpicking here, but think we should change to `AddMemberTests` to avoid confusion with the difference between a (workspace) member and a user.

Reply:
Good point, will change